test/func/docker-compose.yml

```diff
@@ -0,0 +1,381 @@
+version: '3.7'
```

Comment:
What are we trying to achieve with this TTK based functional test stack which we are using in sdk-scheme-adapter?
I don't think this stack can test anything related to mcm client.